apps/condo/domains/user/access/User.js

```diff
@@ -99,6 +99,12 @@ const canAccessToIsAdminField = {
     update: access.userIsAdmin,
 }
 
+const canManageUserType = {
+    read: true,
+    create: canManageUsers,
```

Comment:
🤔 
Should support be banned from creating user? We'll think about it in another PR